js/packages/components/settings/SystemInfo.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect } from 'react'
+import { View, ScrollView, ActivityIndicator } from 'react-native'
+import { Layout, Text } from 'react-native-ui-kitten'
+import { useStyles } from '@berty-tech/styles'
+import { HeaderSettings } from '../shared-components/Header'
+import { ScreenProps, useNavigation } from '@berty-tech/navigation'
+import { Settings } from '@berty-tech/hooks'
+import moment from 'moment'
+
+const SystemInfoItem: React.FC<{ label: string; value: string }> = ({ label, value }) => {
+	const [{ row, padding }] = useStyles()
+	return (
+		<View style={[row.left, padding.top.big]}>
+			<Text>{label + ': '}</Text>
+			<Text>{value}</Text>
+		</View>
+	)
+}
+
+const SystemInfoList: React.FC<{}> = () => {
+	const settings = Settings.useSettings()
+	const [{ padding }] = useStyles()
+	return (
+		<View style={[padding.large]}>
+			{settings?.systemInfo &&
+				Object.entries(settings?.systemInfo).map((value, key) => {
+					if (value[0] === 'startedAt') {
+						return (
+							<View>
+								<SystemInfoItem
+									label='startedAt'
+									value={moment.unix(value[1]).format('MMMM Do YYYY, h:mm:ss a')}
+								/>
+								<SystemInfoItem label='timeAgo' value={moment().from(moment.unix(value[1]))} />
```

Comment:
Cool but generates the string in the opposite direction, should not be "in 4 minutes" but "4 minutes ago"